pkg/agent/core/ngt/handler/grpc/handler_test.go

```diff
@@ -4302,6 +4302,813 @@ func Test_server_MultiInsert(t *testing.T) {
 				},
 			}
 		}(),
+
+		func() test {
+			req := genF32Req(vector.Gaussian, 100, f32VecDim)
+			vid := req.Requests[50].Vector.Id
```

Comment:
why `50`?

Reply:
I intentionally set it to 50 because I want to ensure it to produce the correct error on `req[50]` instead of `req[0]`

Reply:
the answer just I wanted is why you set `50` which is the user-defined number.

> I intentionally set it to 50 because I want to ensure it to produce the correct error on req[50] instead of req[0]
what is the difference between `req[0]` and `re[50]`?

Reply:
I think the test case we designed is to test the result of specific user input. 

The difference between 0 and 50 is that the error produce from `MultiInsert()` is different, in this case I want to test if the error detail is referring `req[50]` instead of `req[0]`. 

If you think 0 is better I can change it to 0.

Reply:
IMO, I'd not like to use a magic number without a specific reason.
If I should choose one from `0` or `50`, I'll choose `0`.
It may affect smaller than `50` when any code changes happen.

Reply:
Thank you for your comment. I will update it to `0`.

Reply:
fixed :)
https://github.com/vdaas/vald/pull/1646/commits/0c4d4c5daa13ff8f35c5f843d3fc9ddaf313a733